src/Codeception/Module/Facebook.php

```diff
@@ -121,11 +123,17 @@ public function _requires()
 
     public function _depends()
     {
-        return ['Codeception\Module\PhpBrowser' => $this->dependencyMessage];
+        return ['Codeception\Lib\Interfaces\Remote|Codeception\Lib\Interfaces\Remote' => $this->dependencyMessage];
     }
 
-    public function _inject(PhpBrowser $browserModule)
+    public function _inject($browserModule)
     {
+        if (!$browserModule instanceof Remote || !$browserModule instanceof Web) {
```

Comment:
And check if passed a browserModule is an instance of `Remote`, if not throw extension as you do it here